prelude/bootstrap.js

```diff
@@ -94,12 +94,15 @@ EXECSTAT.birthtimeMs = EXECSTAT.birthtime.getTime();
 var mountpoints = [];
 
 function insideMountpoint(f) {
+  var util = require('util');
   if (!insideSnapshot(f)) return null;
   var file = normalizePath(f);
   var found = mountpoints
     .map((mountpoint) => {
       var { interior } = mountpoint;
       var { exterior } = mountpoint;
+      if (util.isRegExp(interior) && interior.test(file))
```

Comment:
I'm surprised Prettier didn't complain about this - shouldn't it use brackets?

Reply:
The other `if` statements in this function don't use brackets, but I can add it to this if you want.

Reply:
Nah, I'd rather be consistent 👍 Thank you!